application/src/main/java/net/corda/v5/application/interop/facade/FacadeMethod.java

```diff
@@ -45,21 +48,39 @@ public interface FacadeMethod {
     @NotNull
     FacadeResponse response(TypedParameterValue<?>... parameterValues);
 
+    /**
+     * @return facadeId The [FacadeId] of the owning facade.
+     */
     @NotNull
     FacadeId getFacadeId();
 
+    /**
+     * @return The name of the method
+     */
     @NotNull
     String getName();
 
+    /**
+     * @return The type of the facade method, can be of two types: command or query
```

Comment:
It's nice to explain more here, but what "semantic" types are returned should be explained in `FacadeMethodType`, so I'm fine with `The type of the facade method` or some general explanaition why I would care about different types.